multilabel_classification/pytorch_network_training.ipynb

```diff
@@ -0,0 +1,1064 @@
+      "LOSS: 0.23149096965789795\n",
+      "LOSS: 0.21479015052318573\n",
+      "LOSS: 0.19762173295021057\n",
+      "LOSS: 0.23581306636333466\n",
+      "LOSS: 0.18089541792869568\n",
+      "LOSS: 0.18147560954093933\n",
+      "LOSS: 0.21658293902873993\n",
+      "LOSS: 0.20164594054222107\n",
+      "LOSS: 0.1911771595478058\n",
+      "LOSS: 0.17672832310199738\n",
+      "LOSS: 0.17885778844356537\n",
+      "LOSS: 0.20742546021938324\n",
+      "LOSS: 0.2546771466732025\n",
+      "LOSS: 0.16654500365257263\n",
+      "LOSS: 0.23142722249031067\n",
+      "LOSS: 0.18058699369430542\n",
+      "LOSS: 0.19054488837718964\n",
+      "TRAINING DATA STATS\n",
+      "AVERAGE LOSS: 0.2006054066614681\n",
+      "MULTILABEL accuracy score: 0.5426535584768812\n",
+      "['Eyeglasses', 'Wearing_Earrings', 'Wearing_Hat', 'Wearing_Necklace', 'Wearing_Necktie', 'No_Beard', 'Smiling']\n",
+      "[0.9849912171350861, 0.8584967135086129, 0.9771858000906618, 0.8681436990027198, 0.9359417497733454, 0.9418843495013599, 0.8494730281051677]\n",
+      "\n",
+      "\n",
+      "\n",
+      "training time 352.976446390152\n"
+     ]
+    }
+   ],
+   "source": [
+    "ct = 1\n",
+    "for train_index, test_index in skf.split(df,df['unique_label']):\n",
+    "    if ct!=1:\n",
+    "        model.apply(reset_weights);\n",
+    "    dataset_train = DatasetMultiLabel(df = df.loc[train_index])\n",
+    "    dataset_val = DatasetMultiLabel(df = df.loc[test_index])\n",
+    "    loader_train = create_loader(\n",
+    "        dataset_train,\n",
+    "        input_size=data_config[\"input_size\"],\n",
+    "        batch_size=64,\n",
+    "        is_training=True,\n",
+    "        mean=data_config[\"mean\"],\n",
+    "        std=data_config[\"std\"],\n",
+    "       interpolation=data_config[\"interpolation\"],\n",
+    "    )\n",
+    "    loader_val = create_loader(\n",
+    "        dataset_val,\n",
+    "        input_size=data_config[\"input_size\"],\n",
+    "        batch_size=64,\n",
+    "        is_training=False,\n",
+    "        mean=data_config[\"mean\"],\n",
+    "        std=data_config[\"std\"],\n",
+    "        interpolation=data_config[\"interpolation\"],\n",
+    "\n",
+    "    )\n",
+    "    model.fit(loader_train,loader_val,num_epochs=40)\n",
+    "    checkpoint = torch.load(\"weights_model/model_best.pth.tar\")\n",
+    "    model.load_state_dict(checkpoint['state_dict'])\n",
+    "    pred_val = model.predict_proba(loader_val)\n",
+    "    df_pred = create_df(pred_val,dataset_val)\n",
+    "    df_pred.to_csv(str(ct)+\"_fold.csv\",index=False)\n",
+    "    ct+=1\n",
+    "    \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<module 'os' from '/usr/lib/python3.8/os.py'>"
+      ]
+     },
+     "execution_count": 20,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
```

Comment:
get rid of this?  not sure what it is doing here

Reply:
Done: https://github.com/cleanlab/examples/pull/27/commits/28f62b8528fdc7aec2fd87bca2d5da8e142b7b1e